packages/vue-remark/README.md

```diff
@@ -0,0 +1,151 @@
+# @gridsome/vue-remark
+
+> Create pages with Vue components in markdown. Perfect for building Documentation, Design Systems, Portfolios, Blogs, etc.
+
+## Install
+
+- `yarn add @gridsome/vue-remark`
+- `npm install @gridsome/vue-remark`
+
+## Usage
+
+```js
+module.exports = {
+  plugins: [
+    {
+      use: '@gridsome/vue-remark',
+      options: {
+        typeName: 'MarkdownPage', // required
+        baseDir: './src/pages', // required
+      }
+    }
+  ]
+}
+```
+
+## Example component
+
+```jsx
```

Comment:
This worked for me with a <style> tag where there were no empty lines, but not one where there were empty lines.

Reply:
The latest commit should make it work better :)